pkg/kubeclient/signup.go

```diff
@@ -0,0 +1,40 @@
+package kubeclient
+
+import (
+	crtapi "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+)
+
+const (
+	userSignupResourcePlural = "usersignups"
+)
+
+type userSignupClient struct {
+	crtClient
+}
+
+type UserSignupInterface interface {
+	Get(name string) (*crtapi.UserSignup, error)
+	Create(obj *crtapi.UserSignup) (*crtapi.UserSignup, error)
+}
+
+func (c *userSignupClient) Get(name string) (*crtapi.UserSignup, error) {
```

Comment:
could we add some documentation to our public apis?

Reply:
Done